skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,147 @@
+import numpy as np
+from scipy import ndimage
+
+
+def warp_images(
+    from_points,
+    to_points,
+    images,
+    output_region,
+    interpolation_order=1,
+    approximate_grid=2
+):
+    """Return an array of warped images.
+
+    Define a thin-plate-spline warping transform that warps from the
+    from_points to the to_points, and then warp the given images by
```

Comment:
I believe it would actually be:

* "from the `from_points` points to the `to_points` points" or
* "from `from_points` to `to_points`" and,
* consistently, "the given images" would be "`images`"? 
```